index.js

```diff
@@ -373,62 +373,86 @@
   ];
 
   //  ValidDate :: Type
-  $.ValidDate = $.NullaryType(
+  $.ValidDate = NullaryType(
     'sanctuary-def/ValidDate',
     function(x) { return $.Date.test(x) && !isNaN(x.valueOf()); }
   );
 
+  //  PositiveNumber :: Type
+  $.PositiveNumber = NullaryType(
+    'sanctuary-def/PositiveNumber',
+    function(x) { return $.Number.test(x) && x > 0; }
```

Comment:
> Do you think we might include other, pre-built, environments that provide other convenient groupings?

We could do, though I'm leaning in the other direction. We can't know which types will be relevant in a given context, so there's an argument to be made for requiring types to be selected à la carte. The main role of `$.env` is to make it easy to "set up" the library in the REPL, but `[$.Array, $.Boolean, $.Number, $.String]` would likely perform this role equally well.

Were we to provide pre-built environments, what might we name them and which types might they comprise?